content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -15,17 +15,411 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
-Description here.
+Thanks to the fantastic work of our ECS team, the hotly awaited ["stageless" scheduling RFC](https://github.com/bevyengine/rfcs/blob/main/rfcs/45-stageless.md) has been implemented! But as we all know, plans and implementations (start at [#6587](https://github.com/bevyengine/bevy/pull/6587) by `@maniwani` and [#7267](https://github.com/bevyengine/bevy/pull/7267) by `@alice-i-cecile`) are two different things. Let's take a look at what actually shipped for 0.10.
+
+There's been a lot of changes, but we really do think that ripping off the band-aid now (before any form of stability guarantees) is essential to the health of Bevy's scheduling model going forward.
+
+The [migration path](../../learn/book/migration-guides/0.9-0.10/_index.md) for existing applications won't be trivial, but we've done our best to keep it surprisingly straightforward. Don't sweat it!
+
+## A Single Unified Schedule
+
+Ever wanted to specify that `system_a` runs before `system_b`, only to be met with confusing warnings that `system_b` isn't found because it's in a different stage?
+
+No more! All systems within a single **schedule** are now stored in a single data structure with a global awareness of what's going on.
+
+This simplifies our internal logic, makes your code more robust to refactoring, and allows plugin authors to specify high-level invariants (e.g. "movement must occur before collision checking") without locking themselves in to an exact schedule location.
+
+[!main_schedule_diagram](main_schedule_diagram.svg)
+
+This diagram, made with [@jakobhellermann's `bevy_mod_debugdump` crate](https://github.com/jakobhellermann/bevy_mod_debugdump) shows a simplified version of Bevy's default schedule.
+
+## Configurable System Sets
+
+To support more natural and flexible control over "how are my systems run and scheduled", the idea of a "system set" has been redefined, rolling up the existing "system label" concept into one straightforward but powerful abstraction.
+
+**System sets** are named collections of systems that share a set of **system configuration**: if there are run conditions attached, how they are ordered relative to other systems or sets and so on. This is distributive: Ordering systems relative to a system set applies that ordering to _all_ systems in that set.
+
+Let's jump right in to what this would look like.
+
+```rust
+// System set types are used to provide stable, typed identifiers
+// for groups of systems, allowing external systems to order themselves
+// without being aware of internal details.
+// Each variant of this enum is a distinct system set.
+#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+enum PhysicsSet {
+    Forces,
+    Kinematics,
+    CollisionDetection
+}
+
+app
+    // .with_run_criteria -> .run_if <3
+   .add_system(gravity.in_set(PhysicsSet::Forces).run_if(gravity_enabled))
```

Comment:
Comprehension check from a casual reader: will this run condition apply to any systems added to this set or have I misunderstood the comment above about run conditions and shared configuration?